route/include/route_generator_worker.h

```diff
@@ -153,6 +159,38 @@ namespace route {
          */
         visualization_msgs::MarkerArray compose_route_marker_msg(const lanelet::Optional<lanelet::routing::Route>& route);
 
+        /**
+        * \brief crosstrack_error_check is a function that determines when the vehicle has left the route and reports when a crosstrack error has
+        * taken place
+        * 
+        *  \param msg Msg that contains the vehicle's current position
+        *  \param current_llt The lanelet that the vehicle is currently in
+        *  \param llt_track The crosstrack and downtrack distance of the current lanelet
+        * */
+        bool crosstrack_error_check(const geometry_msgs::PoseStampedConstPtr& msg, lanelet::ConstLanelet current_llt);
+
+        /**
+         * \brief set the crosstrack error counter
+         * 
+         *  \param cte_counter initial value of the counter (set to zero as specified in the route_config_params.yaml file)
+        */
+        void set_CTE_counter(int cte_counter);
```

Comment:
Let's remove this as it is better to initialize it to 0 in the headerfile. Otherwise, it might be unnecessarily confusing.